### PR TITLE
Common instructions how to merge PRs and keep history clean

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,16 @@
 - [How To Build](https://wiki.status.im/Building_status-go)
 - [Notes on Bindings](https://wiki.status.im/Status-go_Binding_notes)
 
+# Notes on Git/GitHub workflow
+
+1. Squash PR before merging.
+   You can do it either with GitHub api or locally if you want to preserve signature.
```
Comment (adambabik) [inline README.md]: We can do be more explicit with `with Github API by merging using "Squash and merge" or locally [...]`.

Also `preserve your signature`.

**Review by adambabik — APPROVED** [review]

**Review by themue — APPROVED** [review]
LGTM when following Adams comment.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,16 @@
...
+   Don't use `Create merge commit` option on GitHub. It pollutes history with meaningless information.
```
Comment (divan) [inline README.md]: Actually, this part is unneeded as Merge button functionality can be restricted via repository configuration. We can restrict any of the available options.

**Review by JekaMas — APPROVED** [review]
Great suggestion!

**Comment by adriacidre** [comment]
Maybe makes more sense to have this notes on [CONTRIBUTING.md](https://github.com/status-im/status-go/blob/develop/.github/CONTRIBUTING.md) and add a link to the main README.

File: README.md (hunk 2 above)
Comment (mandrigin) [reply README.md]: @divan can you do that now? that's a nice thing, I don't see us ever wanting to have merge commits in the tree. I don't seem to have rights to tweak these settings now.

**Comment by mandrigin** [comment]
@dshulyak what should happen to this PR?
It's all good suggestion, but I +1 to @adriacidre to move it CONTRIBUTING.md. Let's do that, fix the feedback and merge.

**Review by adriacidre — APPROVED** [review]

**Comment by dshulyak** [comment]
@mandrigin moved to CONTRIBUTING.md

**Pull request merged**